clients/venice-admin-tool/src/main/java/com/linkedin/venice/Command.java

```diff
@@ -447,7 +447,8 @@ public enum Command {
   ),
   EXECUTE_DATA_RECOVERY(
       "execute-data-recovery", "Execute data recovery for a group of stores",
-      new Arg[] { RECOVERY_COMMAND, STORES, SOURCE_FABRIC }, new Arg[] { EXTRA_COMMAND_ARGS, DEBUG, NON_INTERACTIVE }
+      new Arg[] { URL, RECOVERY_COMMAND, STORES, DEST_FABRIC, DATETIME },
```

Comment:
SOURCE_FABRIC is useful, I think we should add DEST_FABRIC as a new Arg, but not replacing the existing SOURCE_FABRIC

Reply:
Re-added this parameter